src/modules/add-task.js

```diff
@@ -0,0 +1,134 @@
+
+    if (storedList) {
+      list = JSON.parse(storedList);
+      index = list.length === 0 ? 0 : list.length;
+    } else {
+      list = [];
+    }
+
+    const store1 = {
+      index,
+      description: taskInput.value,
+      completed,
+    };
+    list.push(store1);
+    localStorage.setItem('todo-list', JSON.stringify(list));
+    showTodo();
+  }
+});
+
+window.editItem = (index) => {
+  const editBtn = document.getElementById(`edit${index}`);
+  const deletebtn = document.getElementById(`delete${index}`);
+
+  deletebtn.style.display = 'block';
+  editBtn.style.display = 'none';
+  const mainItem = document.getElementById(`item${index}`);
+  mainItem.removeAttribute('readonly');
+  const { length } = mainItem.value;
+  mainItem.setSelectionRange(length, length);
+};
+
+window.removeItem = (index) => {
+  const editBtn = document.getElementById(`edit${index}`);
+  const deletebtn = document.getElementById(`delete${index}`);
+
+  deletebtn.style.display = 'none';
+  editBtn.style.display = 'block';
+  const storedData = localStorage.getItem('todo-list');
+  list = JSON.parse(storedData);
+  list.splice(index, 1);
+  for (let i = 0; i < list.length; i += 1) {
+    list[i].index = i;
+  }
+  localStorage.setItem('todo-list', JSON.stringify(list));
+  showTodo();
+};
+
+const saveTaskOnBlur = (event) => {
+  const taskItem = event.target;
+  const itemId = taskItem.getAttribute('id');
+  const index = parseInt(itemId.replace('item', ''), 10);// radi
+  list[index].description = taskItem.value;
+  localStorage.setItem('todo-list', JSON.stringify(list));
+  const editBtn = document.getElementById(`edit${index}`);
+  const deletebtn = document.getElementById(`delete${index}`);
+
+  deletebtn.style.display = 'none';
+  editBtn.style.display = 'block';
+};
+taskBox.addEventListener('blur', saveTaskOnBlur, true);
+
+window.CheckMe = (index) => {
+  const CheckCheck = document.getElementById(`check${index}`);
+  if (CheckCheck.checked === true) {
+    const storedData = localStorage.getItem('todo-list');
+    list = JSON.parse(storedData);
+    list[index].completed = true;
+    localStorage.setItem('todo-list', JSON.stringify(list));
+    showTodo();
+  } else {
+    const storedData = localStorage.getItem('todo-list');
+    list = JSON.parse(storedData);
+    list[index].completed = false;
+    localStorage.setItem('todo-list', JSON.stringify(list));
+    showTodo();
+  }
+};
+
+window.onload = () => {
+  if (localStorage.getItem('todo-list')) {
+    list = JSON.parse(localStorage.getItem('todo-list'));
+  }
+  showTodo();
+};
+
+ClearAll.addEventListener('click', () => {
+  const storedData = localStorage.getItem('todo-list');
+  list = JSON.parse(storedData);
+  const AfterCleared = list.filter((element) => element.completed === false);
+  list = AfterCleared;
+  for (let i = 0; i < list.length; i += 1) {
+    list[i].index = i;
+  }
+  localStorage.setItem('todo-list', JSON.stringify(list));
+  showTodo();
+});
```

Comment:
Consider breaking your code into different sections and using as many ES6 modules as you can as this will make your code more readable and maintainable in larger projects